cmd/libs/go2idl/parser/parse.go

```diff
@@ -316,6 +319,12 @@ func (b *Builder) priorCommentLines(pos token.Pos) string {
 	return ""
 }
 
+func tcFuncNameToName(in string) types.Name {
+	name := strings.TrimLeft(in, "func ")
+	nameParts := strings.Split(name, "(")
+	return tcNameToName(nameParts[0])
```

Comment:
I think this won't happen. Go doesn't allow defining an anonymous function at package level.